devel/udrone/Makefile

```diff
@@ -0,0 +1,34 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=udrone
+PKG_RELEASE:=1
+PKG_VERSION:=839eee4c6fbb85d17528cfd23d5c9cde902f622d
```

Comment:
This might break opkg update functionality. I think opkg works alphabetically when it comes to new versions.

This is typically solved by using PKG_SOURC_DATE elsewhere.

Reply:
Added.